docker-compose/withPostgresAndWorker/docker-compose.yml

```diff
@@ -67,6 +66,8 @@ services:
 
   n8n-worker:
     <<: *shared
-    command: worker
+    command: n8n worker
```

Comment:
On any recent version of n8n, this will most definitely exit the container with the message `Error: command n8n not found`